devai-cli/src/devai/commands/rag/load.py

```diff
@@ -0,0 +1,127 @@
+import click
+from google.cloud import aiplatform
+import os
+import shutil
+from pathlib import Path
+from git import Repo
+
+from langchain_community.document_loaders import GitLoader
+from langchain.text_splitter import RecursiveCharacterTextSplitter
+from langchain_google_vertexai import VertexAIEmbeddings
+from langchain_community.vectorstores import Chroma
+
+
+def load_docs(repo_path, branch, local_dir, allowed_extensions):
```

Comment:
this load everything in the repo that is allowlisted in the extensions list, right? not just docs.

Reply:
correct